Project.toml

```diff
@@ -11,6 +11,7 @@ Images = "916415d5-f1e6-5110-898d-aaa5f9f070e0"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
```

Comment:
Pkg can be removed then?

Reply:
haven't fully removed `p7zip_jll` usage yet, just the minimum to make the tests pass.